### DISCUSSION TITLE
Spare PCB Boards for DL

### DISCUSSION THREAD

**Discussion by DK4FT** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#322] [General]
Hi Group,

I have 2 spare V1 Boards available.
Cost for each is €2 + €1 for shipping inside Germany.
![20220705_120301](https://user-images.githubusercontent.com/62115413/177304603-320135a2-1f1a-49ad-9c59-0268aa103c16.jpg)

another ready build board = sold

**Comment by keitelmaxi** [comment]
Hi, ich bin an dem fertigen Board interessiert. Für welche Pumpe hast du die benutzt?
Kontaktiere mich gerne unter github-whirlpool@keitelmaxi.de

**Reply by DK4FT** [reply]
Hi, das Board war noch nicht im Einsatz.
Ich habe zwei gebaut, ein baugleiches habe ich in einem Layz Honolulu im Einsatz:
- pre2021
- 6-pin

Der NodeMCU von diesem Set ist noch "jungfräulich" und unprogrammiert...

**Reply by keitelmaxi** [reply]
Hi, dann würde ich dir das Board gerne zum oben genannten Preis abnehmen 😁
Grüße

**Reply by DK4FT** [reply]
Ok, verkauft, details per Email.

**Reply by Deefjes** [reply]
kann ich auch eine PCB kaufen? Ich lebe in den Niederlanden.

avanox@outlook.com

**Comment by heijjan** [comment]
Hi, would you be interested in sending one to Finland if available?

**Comment by DK4FT** [comment]
I am sorry, all units are sold...
-------- Ursprüngliche Nachricht --------Von: heijjan ***@***.***> Datum: 27.08.22  09:44  (GMT+01:00) An: visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA ***@***.***> Cc: Alex ***@***.***>, Author ***@***.***> Betreff: Re: [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA] Spare PCB
  Boards for DL (Discussion #322) 
Hi, would you be interested in sending one to Finland if available?
